.github/workflows/mvn-verify-check.yml

```diff
@@ -165,7 +167,9 @@ jobs:
         id: generateCacheKey
         run: |
           set -x
-          cacheKey="${{ runner.os }}-maven-scala213-${{ hashFiles('**/pom.xml') }}-${{ github.event.pull_request.base.ref }}-$(date +'%Y-%m-%d')"
+          jniTimestamp=$(. .github/workflows/mvn-verify-check/get-artifact-timestamp.sh spark-rapids-jni)
+          privateTimestamp=$(. .github/workflows/mvn-verify-check/get-artifact-timestamp.sh rapids-4-spark-private_2.13)
```

Comment:
This already looks good. 

Optional nits:
- we could return a key reflecting both artifacts in a single invocation 
- it is more accurate to use the `sha1` field of the response than the timestamp because in the end it is about whether the artifact changed or not and the timestamp is a secondary indicator. 

So the single invocation  of the scrfipt script could simply return a concatenation of spark-rapdis-jni and private sha1 field.  If we have to limit the number of characters used in the key the script can md5sum the concatenation for the final output of the script.

Reply:
Thanks gera.

I've modified the script to return md5 hash value of concatenation of jni and private sha1.